card/mod/01_core/spec/set/all/trash_spec.rb

```diff
@@ -31,4 +30,30 @@
     end
   end
 
+  it 'deletes children' do
+    Card::Auth.as_bot do
+      name = 'born to die'
+      card = Card.create! name: name, '+sub' => 'a subcard'
+      expect(Card["#{name}+sub"]).to be
+      card.delete
+      expect(Card["#{name}+sub"]).not_to be
+    end
+  end
+
+  it 'deletes account of user' do
+    Card::Auth.as_bot do
+      @signup = Card.create!(
+        name: 'born to die', type_id: Card::SignupID,
+        '+*account' => { '+*email' => 'wolf@wagn.org', '+*password' => 'wolf' }
+      )
+      @signup.update_attributes!({})
+    end
+    Card::Cache.reset_global
+
+    Card::Auth.as_bot do
+      expect(Card.search :right=>'*account').not_to be_empty
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.